addons/project/static/tests/project_task_groupby_tests.js

```diff
@@ -67,7 +67,7 @@ QUnit.module("Project", (hooks) => {
             views: [[false, "kanban"]],
         });
 
-        assert.strictEqual(target.querySelector(".o_column_title").innerText, "🔒 Private (1)");
+        assert.strictEqual(target.querySelector(".o_column_title").innerText, "🔒 Private\n1");
```

Comment:
Why do we have an additional blank line, now? 🤔 🧐

Reply:
:shrug: